app/lib/notifications.dart

```diff
@@ -1,15 +1,57 @@
 import 'package:flutter_local_notifications/flutter_local_notifications.dart';
 import 'package:timezone/timezone.dart' as tz;
```

Comment:
```suggestion
import 'package:timezone/timezone.dart';
```

You might want to remove `tz` in this file to make our source code less sophisticated.